obspy/core/tests/test_utcdatetime.py

```diff
@@ -31,7 +31,9 @@ def test_from_string(self):
         self.assertEqual(dt, UTCDateTime(2009, 7, 1, 12, 12, 12))
         dt = UTCDateTime("19700101")
         self.assertEqual(dt, UTCDateTime(1970, 1, 1, 0, 0))
-        # non ISO8601 strings should raise an exception
+        dt = UTCDateTime("1970/01/01 12:23:34")
```

Comment:
we only support YMD never YDM or other combinations (except ISO8101 stuff) - otherwise things get complicated, but maybe we should state that somewhere